README.md

```diff
@@ -35,8 +40,10 @@ or apt to install these components if they're self-hosted.
 
 ### What splunk affects
 
-* Installs the Splunk/Forwarder package and manages their config files. It does not purge them by default.
-* The module will set up both Splunk and Splunkforwarder to run as the 'root' user on POSIX platforms.
+* Installs the Splunk/Forwarder package and manages their config files. It does
+  not purge them by default.
+* The module will set up both Splunk and Splunkforwarder to run as the 'root'
+  user on POSIX platforms.
```

Comment:
IMO we should honour the linter here. My vim makes automatic linebreaks after X chars, that needs to be configured but works fine.

@crayfishx which parts do you recognize as PITA?